### ISSUE TITLE
Add namespaces support

### ISSUE THREAD

**Issue by vitaliy-bobrov** [vitaliy-bobrov/angular-icon#1]
Create service that will be integrated and used to separate icons by namespaces.

**Comment by JipSterk** [comment]
I would like to help you with this. could you explain the idea a bit more?

**Comment by vitaliy-bobrov** [comment]
@JipSterk, yes it'll be great if you could help. The idea is to create so.ething similar to namespaces in angular-material, https://material.angular.io/components/icon/overview
